.github/workflows/main.yml

```diff
@@ -94,10 +94,10 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
-    - name: Install tox
+    - name: Install tox setuptools_scm
```

Comment:
Should we add a note that this can be removed once we drop Python 2.7 and 3.5?

Reply:
for now not, until the packaging upstream  stabilizes better its a good idea to leave it in for the next setuptools interaction issue